tests/foreman/cli/test_bootdisk.py

```diff
@@ -90,3 +90,48 @@ def _finalize():
     # Check if full-host bootdisk can be downloaded.
     bootdisk = module_target_sat.cli.Bootdisk.host({'host-id': host['id'], 'full': 'true'})
     assert 'Successfully downloaded host disk image' in bootdisk['message']
+
+
+def test_positive_bootdisk_subnet_download(module_location, module_org, module_target_sat):
+    """Verify satellite is able to download subnet disk image.
+
+    :id: ed6da37f-ba3f-43c9-9169-40c818e42136
+
+    :steps:
+        1. Create a subnet
+        2. Download the subnet disk image
+        3. Check if subnet disk image is downloaded properly
+
+    :expectedresults: subnet disk image is downloaded properly
+
+    :Verifies: SAT-23549
```

Comment:
Shouldn't there also be BlockedBy ? so that the test is skipped until the jira is resolved?

Reply:
The issue is resolved already not sure when but the BZ wasn't moved to ON_QA. I've closed the BZ with a comment